src/services/email/smtp.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
Use 
`nest generate service services/email/smtp`
It should create smtp folder too.

Reply:
The smtp.service files have been moved to the correct folder structure now, inside services/email/smtp